<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DNS Spoofer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_arp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>261</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Arp Poison</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_router">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.0.100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_victim">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.0.16</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_router">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Router IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_vivtim">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Victim IP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_poison">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Poison</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_refresh_targets">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_target_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Target List</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_target_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>241</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_dns">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>261</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>DNS Spoof</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_target">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>milliways.bcit.ca</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_redirect">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.0.112</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_target">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Target Domain</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_redirect">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Redirect IP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_spoof">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Spoof</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_refresh_spoof_list">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_spoof_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Spoof List</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_spoof_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>241</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
